templates/HAPROXY_BACKEND_HTTP_HEALTHCHECK_OPTIONS

```diff
@@ -0,0 +1,2 @@
+  option  httpchk GET {healthCheckPath}
```

Comment:
synced with @brndnmtthws on this requirement - the check should be done from the LB because there could be a network partition where marathon can access the service but the LB can't